polling_stations/apps/file_uploads/management/commands/check_pending.py

```diff
@@ -0,0 +1,13 @@
+from django.core.management.base import BaseCommand
+from file_uploads.models import UploadQuerySet
+
+
+class Command(BaseCommand):
+    def handle(self, *args, **kwargs):
+        qs = UploadQuerySet.pending_upload_qs(self)
+        if len(qs) == 0:
+            pass
+        else:
+            for upload in qs:
+                upload.send_error_email()
+        return qs
```

Comment:
I don't think there's any harm in this, but it looks like the return value is mostly there for testing. In general it would be better to test the `pending_upload_qs` method on the `Upload` model's test case. The obvious issue with that is there's no tests for the `Upload` model. (:raised_hand_with_fingers_splayed: guilty). So it's fine like this for now, but something to come back to.

Reply:
I was tempted to add more tests but didn't want to go too far out of scope. Happy to do that in another PR and adjust this method as suggested then.